qiskit/utils/deprecation.py

```diff
@@ -23,7 +189,7 @@ def deprecate_arguments(
     *,
     since: Optional[str] = None,
 ):
-    """Decorator to automatically alias deprecated argument names and warn upon use.
+    """Deprecated. Instead, use `@deprecate_arg`.
```

Comment:
I'm not sure how we should deprecate this -- should we use `deprecate_func`? I think that will cause noise for end-users?